plugins/python-build/share/python-build/pyston-0.6.1

```diff
@@ -0,0 +1,23 @@
+case "$(pyston_architecture 2>/dev/null || true)" in
+"linux64" )
+  install_package "pyston-0.6.1-linux64" "https://github.com/dropbox/pyston/releases/download/v0.6.1/pyston-0.6.1-linux64.tar.gz#b09516d8e421cb004ef39d832bae600face4ecf80d9791469fb2c026bcf1faae" "pyston" verify_py27
```

Comment:
I found these 2 lines in the Stackless 3.4.2 definition which could help with "openssl-1.0.2k" and "readline-6.3". Should I use these instead?

https://github.com/pyenv/pyenv/blob/d1ff898a2bfbe44d74bf92c2b6c9f7652b1d3aaa/plugins/python-build/share/python-build/stackless-3.4.2#L2